.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,14 @@
 {
-  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  "image": "mcr.microsoft.com/devcontainers/universal:2.3.1",
+  "hostRequirements": {
+    "cpus": 4,
+    "memory": "8gb",
+    "storage": "32gb"
+  },
   "features": {
-    "ghcr.io/balazs23/devcontainers-features/bazel:1": {}
+    "ghcr.io/balazs23/devcontainers-features/bazel:1.0.1": { 
+      "bazelisk": "v1.15.0"
+    }
   },
-  "onCreateCommand": ["sudo", "ln", "-s", "bazelisk", "/usr/local/bin/bazel"]
+  "containerEnv": { "USE_BAZEL_VERSION": "6.1.1" }
```

Comment:
Why have this? it's redundant with the `.bazelversion` file in the repo

Reply:
It only does that if it doesn't see `.bazelversion` file.

Reply:
I guess to be clear - I assumed that pre-loading bazel at container startup would be a slightly nicer experience, but it doesn't behave as I thought it did. That way it wouldn't have to re-download bazel 6.1.1 when you run `bazel` from the workspace root, it could just grab it from a local cache. In hindsight though, a bad assumption.